demos/dotnet/dotnet/dotnet.cs

```diff
@@ -64,6 +86,7 @@ static void PrintUsage()
         string appName = Environment.GetCommandLineArgs()[0];
         Console.WriteLine("{0} [/log] - compiles sources in current direcotry. optionally logs diagnostics info.", appName);
         Console.WriteLine("{0} /target:library - compiles sources in current directory into dll.", appName);
+        Console.WriteLine("{0} /recurse:<wildcard> - compiles sources in current directory and subdirectories according to the wildcard specifications.", appName);
```

Comment:
Hopefully packages directory will never have cs files :-) And more seriously, I think we need to filter out subdirectories created by the tool: packages, bin, tools.